tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "outDir": "dist",
     "module": "esnext",
     "target": "es5",
-    "lib": ["es6", "dom", "es2019"],
+    "lib": ["es6", "dom", "es2019", "ES2020"],
```

Comment:
just curious, which es2020 syntax are you using? we should make sure this doesn't impact our supported browsers (webpack probably transpiles it away but not sure)

Reply:
(I realized it was for `Promise.allSettled`, which was still giving me errors, so I added `"ES2020.Promise"` which solved it)